Dr Elephand is not starting

Hi Guys, we have been trying to install Dr. Elephant and after a lot of troubleshooting we are stuck in this part:

[root@ip-172-31-37-252 bin]# ./start.sh
Using config dir: /usr/dr-elephant/app-conf
Using config file: /usr/dr-elephant/app-conf/elephant.conf
Reading from config file...
db_url:  localhost
db_name:  drelephant
db_user:  root
http port:  8081
This is hadoop2.x grid. Add Java library path: /lib/native
Starting Dr. Elephant ....
Dr. Elephant started.

but the process never start and looking in the logs we found this:


2016-04-22 20:34:50,374 - [ERROR] - from com.jolbox.bonecp.hooks.AbstractConnectionHook in main
Failed to obtain initial connection Sleeping for 0ms and trying again. Attempts left: 0. Exception: java.net.ConnectException: Conn
ection refused.Message:Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
/hadoop/dr-elephant-2.0.3-SNAPSHOT/logs

please let us know if somebody has been experience this issue or have any idea.

--- a/project/Dependencies.scala
+++ b/project/Dependencies.scala
@@ -40,6 +40,21 @@ object Dependencies {
     sparkVersion = System.getProperties.getProperty(SPARK_VERSION)
   }
 
+  val sparkExclusion = if (sparkVersion >= "1.5.0") {
+    "org.apache.spark" % "spark-core_2.10" % sparkVersion excludeAll(
+      ExclusionRule(organization = "com.typesafe.akka"),
+      ExclusionRule(organization = "org.apache.avro"),
+      ExclusionRule(organization = "org.apache.hadoop"),
+      ExclusionRule(organization = "net.razorvine")
+    )
+  } else {
+    "org.apache.spark" % "spark-core_2.10" % sparkVersion excludeAll(
+      ExclusionRule(organization = "org.apache.avro"),
+      ExclusionRule(organization = "org.apache.hadoop"),
+      ExclusionRule(organization = "net.razorvine")
+    )
+  }
+
   // Dependency coordinates
   var requiredDep = Seq(
     "com.google.code.gson" % "gson" % gsonVersion,
@@ -50,16 +65,10 @@ object Dependencies {
     "org.apache.hadoop" % "hadoop-common" % hadoopVersion % "compileonly",
     "org.apache.hadoop" % "hadoop-common" % hadoopVersion % Test,
     "org.apache.hadoop" % "hadoop-hdfs" % hadoopVersion % "compileonly",
-    // TODO: Cleanup Spark dependencies
-    "org.apache.spark" % "spark-core_2.10" % sparkVersion excludeAll(
-            ExclusionRule(organization = "org.apache.avro"),
-            ExclusionRule(organization = "org.apache.hadoop"),
-            ExclusionRule(organization = "net.razorvine")
-            ),
     "org.codehaus.jackson" % "jackson-mapper-asl" % jacksonMapperAslVersion,
     "org.jsoup" % "jsoup" % jsoupVersion,
     "org.mockito" % "mockito-core" % "1.10.19"
-  )
+  ) :+ sparkExclusion 
 
   var dependencies = Seq(javaJdbc, javaEbean, cache)
   dependencies ++= requiredDep

